src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import React, { useCallback, useEffect, useRef } from 'react';
+
+type Props = {
+  error: string,
+  setError: React.Dispatch<React.SetStateAction<string>>,
+  hasError: boolean,
+  setHasError: React.Dispatch<React.SetStateAction<boolean>>,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  error,
+  setError,
+  hasError,
+  setHasError,
+}) => {
+  const resetError = useCallback(
+    () => {
+      setError('');
+      setHasError(false);
+    },
+    [error],
+  );
+
+  const timerRef = useRef<NodeJS.Timer>();
+
+  useEffect(() => {
+    if (hasError) {
+      timerRef.current = setTimeout(resetError, 3000)
+    } else {
+      clearTimeout(timerRef.current)
+    }
+  }, [hasError]);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        {
+          hidden: !hasError,
+        },
```

Comment:
check here or above about what you get here if you have an error 
![image](https://user-images.githubusercontent.com/31005050/207416650-b397d0c0-d4eb-4474-a2f1-1e5beed29a7d.png)

Reply:
I have no idea why they are showing in the demo link only in my browser there are no error window. Anyways they are related to part 2 so I will fix them during making part 2